src/test/java/zowe/client/sdk/rest/ZoweRequestTest.java

```diff
@@ -220,10 +220,10 @@ public void tstUrlConstructionWithoutBasePathSuccess() {
     }
 
     @Test
-    public void tstUrlConstructionWithInvalidBasePathFailure() {
+    public void tstUrlConstructionWithInvalidHostNameFailure() {
         // Create a connection and set an empty base path
         final ZosConnection connection = ZosConnectionFactory
-                .createBasicConnection("test.host", "443", "user", "password", "frank//");
+                .createBasicConnection("test.host:123foo", "443", "user", "password", "frank///");
```

Comment:
I think here we can just use:
.createBasicConnection("test.host:123foo", "443", "user", "password");